libcloud/compute/drivers/ec2.py

```diff
@@ -5659,7 +5659,7 @@ def __init__(self, key, secret=None, secure=True, host=None, port=None,
         self.region_name = region
         self.api_name = details['api_name']
         self.country = details['country']
-        self.signature_version = details.pop('signature_version',
+        self.signature_version = details.get('signature_version',
```

Comment:
this was opened as another PR and merged - https://github.com/apache/libcloud/pull/523 - forgot to close it!